ops/mpt-switch-test/README.md

```
@@ -0,0 +1,187 @@
+# MPT Switch Test
+
+Test the client switching logic of sequencer node and sentry node at MPT upgrade time.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)